code/modules/reagents/chemistry/holder.dm

```diff
@@ -494,6 +535,10 @@
 		var/datum/reagent/R = reagent
 		switch(react_type)
 			if("LIVING")
+				var/check = reaction_check(A, R)
+				if(!check)
+					continue
+				R.reaction_mob(A, method, R.volume * volume_modifier)
```

Comment:
Riddle me this: You add a `reaction_mob()` call here, but this isn't a proc you added in this PR. Does that mean there's another `reaction_mob()` being called somewhere else? Seems like a bad idea to me.

Reply:
I guess it is, in theory, being called twice on touch then, isn't it? Hmm.

I should probably remove this then.